OpenSim/Examples/OptimizationExample_Arm26/OptimizationExample.cpp

```diff
@@ -160,7 +160,7 @@ int main()
         //Optimizer opt(sys, InteriorPoint);
 
         // Specify settings for the optimizer
-        opt.setConvergenceTolerance(0.1);
+        opt.setConvergenceTolerance(0.11);
```

Comment:
Haha of course- duh. Sorry about that. Yes they are different indeed. Well I think it is a separate issue to refine the tests so they run faster and still produce consistent (expected) results on the different platforms.